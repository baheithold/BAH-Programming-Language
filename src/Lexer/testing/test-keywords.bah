# This file tests valid keywords


if
else
for
while
function
var
return
break
continue
class
lambda
void
true
false
null
