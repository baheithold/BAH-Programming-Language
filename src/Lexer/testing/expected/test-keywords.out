IF_KEYWORD
ELSE_KEYWORD
FOR_KEYWORD
WHILE_KEYWORD
FUNCTION_KEYWORD
VAR_KEYWORD
RETURN_KEYWORD
BREAK_KEYWORD
CONTINUE_KEYWORD
CLASS_KEYWORD
LAMBDA_KEYWORD
VOID_KEYWORD
BOOLEAN_TYPE
BOOLEAN_TYPE
NULL_TYPE
END_OF_INPUT
